frontend/src/components/CourseModalOverview.js

```diff
@@ -106,18 +106,23 @@ const CourseModalOverview = ({ setFilter, filter, setSeason, listing }) => {
     },
   });
   // Hold list of evaluation dictionaries
-  let evaluations = [];
+  let evaluations = useMemo(() => {
+    return [];
+  }, []);
```

Comment:
this seems kinda sus - all the logic to compute the evaluations should be inside here right

Reply:
oh yah im tweaking. just moved `let evaluations = []` into the useMemo for items